Source/DataModels/Scene/Scene.cpp

```diff
@@ -605,6 +605,24 @@ GameObject* Scene::SearchGameObjectByID(UID gameObjectID) const
 	return nullptr;
 }
 
+std::vector<GameObject*> Scene::SearchGameObjectByTag(const std::string& gameObjectTag) const
+{
+	std::vector<GameObject*> tagGameObjects;
+	for (int i = 0; GameObject* gameObject : sceneGameObjects)
+	{
+		if (gameObject && gameObject->CompareTag(gameObjectTag))
+		{
+			tagGameObjects.push_back(gameObject);
+		}
+		i = i + 1;
+	}
+	if (tagGameObjects.size() == 0)
```

Comment:
```suggestion
	if (tagGameObjects.empty())
```
https://cplusplus.com/reference/vector/vector/empty/

Reply:
True, that's better, thanks